lib/segment.js

```diff
@@ -48,20 +48,11 @@ Segment.prototype.allBlocks = function(){
   return result;
 };
 
-Segment.prototype.colorAmount = function (color) {
```

Comment:
Was there a test for this that is now breaking? I'm assuming not - but just checking